docs/modules/servers/pages/distributed/benchmark/james-benchmark.adoc

```diff
@@ -0,0 +1,100 @@
+= Distributed James Server benchmark
+:navtitle: James benchmarks
+
+This document provides benchmark methodology and basic performance of Distributed James as a basis for a James administrator who
+can test and evaluate if his Distributed James is performing well.
+
+It includes:
+
+* A sample Distributed James deployment topology
+* Propose benchmark methodology
+* Sample performance results
+
+This aims to help operators quickly identify performance issues.
+
+== Sample deployment topology
+
+We deploy a sample topology of Distributed James with these following components:
+
+- Distributed James: 3 Kubernetes pods, each pod has 2 OVH vCore CPU and 4 GB memory limit.
+- Apache Cassandra 4 as main database: 3 nodes, each node has 8 OVH vCores CPU and 30 GB memory limit (OVH b2-30 instance).
+- OpenDistro 1.13.1 as search engine: 3 nodes, each node has 8 OVH vCores CPU and 30 GB memory limit (OVH b2-30 instance).
+- RabbitMQ 3.8.17 as message queue: 3 Kubernetes pods, each pod has 0.6 OVH vCore CPU and 2 GB memory limit.
+- OVH Swift S3 as an object storage
+
+== Benchmark methodology and base performance
+
+=== Provision testing data
+
+Before doing the performance test, you should make sure you have a Distributed James up and running with some provisioned testing
+data so that it is representative of reality.
+
+Please follow these steps to provision testing data:
+
+* Prepare James with a custom `mailetcontainer.xml` having Random storing mailet. This help us easily setting a good amount of
+provisioned emails.
+
+Add this under transport processor
+----
+<mailet match="All" class="RandomStoring"/>
+----
+
+* Modify https://github.com/apache/james-project/tree/master/server/apps/distributed-app/docs/modules/ROOT/pages/benchmark/provision.sh[provision.sh]
```

Comment:
```suggestion
* Modify https://github.com/apache/james-project/tree/master/docs/modules/servers/pages/distributed/benchmark/provision.sh[provision.sh]
```

Reply:
Outdated because the file has been moved.

We can fix later IMO...

Reply:
Good catch. There was so many links to adapt with the move that it's highly possible a few have been forgotten :) 

Will take care of it